src/pages/CreatePersona/CreateCard/CreateCard.tsx

```diff
@@ -1,38 +1,161 @@
-import React, { FC, useCallback, useEffect } from 'react';
-import { useLazyQuery } from '@apollo/react-hooks';
+import React, { ChangeEvent, FC, useState, useCallback } from 'react';
+import { useMutation, useQuery } from '@apollo/react-hooks';
 import { useHistory } from 'react-router-dom';
+import { useFormik } from 'formik';
 
-import { GET_CARD, GetCardType } from 'global/graphqls/Persona';
+import { GET_CARD, GetCardType, UPDATE_CARD } from 'global/graphqls/Persona';
 
 import { TopNav } from 'components/TopNav/TopNav';
 import { WideButton } from 'components/Button/WideButton';
-import { PageWrapperFromBottom } from 'components/PageWrapper';
+import { PageWrapperSpaceBetween } from 'components/PageWrapper';
+import { InfoCard } from 'components/InfoCard/InfoCard';
+import { CardType, cardSchema } from 'global/ApolloLinkState/namespace';
+import { Card } from 'components/Card/Card';
+import { BackgroundPlaceholder } from 'components/BackgroundPlaceholder/BackgroundPlaceholder';
+import { PersonaCircle, PersonaCircleWrapper } from 'components/PersonaCircle/PersonaCircle';
+import { Flex } from 'components/FlexBox/FlexBox';
+import { FileInput } from 'components/FileInput/FileInput';
+import { CropperWidget, ImageRef } from 'components/CropperWidget/CropperWidget';
+import { Stepper } from 'components/Stepper';
+
+import { CardBody, CardDescription, CardName, EditButton } from './CreateCard.styles';
+
+const cardInitialValues: CardType = {
+  name: '',
+  description: '',
+  avatar: '',
+  background: '',
+};
+
+const initialState: ImageRef = {
+  blobUrl: '',
+  fieldName: '',
+  blob: null,
+};
 
 export const CreateCard: FC = () => {
-  const [getCard, { data }] = useLazyQuery<GetCardType>(GET_CARD);
+  const [updateCard] = useMutation<GetCardType>(UPDATE_CARD);
+  const { data } = useQuery<GetCardType>(GET_CARD);
+  const [imageRef, setImageRef] = useState<ImageRef>(initialState);
   const history = useHistory();
+  const initialValues = data ? data.persona.card : cardInitialValues;
+
+  const { values, setFieldValue, setFieldTouched, handleSubmit, errors, isValid } = useFormik<CardType>({
+    initialValues,
+    onSubmit: formValues => {
+      updateCard({
+        variables: {
+          card: formValues,
+        },
+      }).then(() => {
+        history.push({
+          pathname: 'page',
+        });
+      });
+    },
+    validationSchema: cardSchema,
+  });
+
+  const { name, description, avatar, background } = values;
 
-  const onNextClick = useCallback((): void => {
-    history.push({
-      pathname: 'page',
-    });
-  }, [history]);
+  const onAvatarChange = useCallback(
```

Comment:
I don't think we should wrap this method in useCallback. Maybe onCrop could be wrapped as it might be worth to avoid  widget rerenders. 
Reference: https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
I got rid of the useCallback